packages/teams-js/src/public/app.ts

```diff
@@ -814,6 +814,10 @@ export namespace app {
 
   /**
    * Notifies the frame that app has loaded and to hide the loading indicator if one is shown.
+   * 
+   * @remarks
+   * For information about mobile behavior, see the [Content pages](https://learn.microsoft.com/microsoftteams/platform/tabs/how-to/create-tab-pages/content-page?tabs=teamsjs-v2#show-a-native-loading-indicator) article.
```

Comment:
>For information about mobile behavior [](http://example.com/codeflow?start=5&length=37)

What should we expect developers to see when they go to that URL? Here is where I think I would be confused: is that page telling me that... I shouldn't call methods like `notifyAppLoaded()` when I'm on mobile?

I think the answer is "No, your app should always call these methods and you don't need to write special code, but just know that the loading indicator behavior may be different on mobile."  Is that correct? If so it's not totally clear from these comments or that doc page.